foreign_cc/private/framework.bzl

```diff
@@ -209,6 +209,7 @@ CC_EXTERNAL_RULE_ATTRIBUTES = {
         cfg = "exec",
         default = [],
     ),
+    "_android_constraint": attr.label(default = Label("@platforms//os:android")),
```

Comment:
I chose to only add this one for now since it's a bit of boilerplate to add more, but I can if folks want to